### DISCUSSION TITLE
dotenv not updating?

### DISCUSSION THREAD

**Discussion by zack-later** [jef/streetmerchant#1703] [Help]
Running into a weird issue I'm not sure what im doing wrong. I made of copy of the example file as dotenv. Set up only the stores and series i wanted but when i run it im getting all the series and a few more stores that aren't even in that file. Its gotta be getting it somewhere is there something else i need to find and delete?

**Comment by zack-later** [comment]
Ok well of course as soon as i ask the question i figure it out. I still had the project saved as nvidia-snatcher from back in the day. I deleted those old files and re cloned the streetmerchant project. Updated the dotenv file and everything seems to be running as expected!

**Reply by TimothyKool** [reply]
Can I ask how you're editing the "dotenv" file? I've been setting the items I want to "true" (tried both with and without quotation marks) but I'm still getting results from all products and stores.
